### ISSUE TITLE
:Map panic

### ISSUE THREAD

**Issue by justinmk** [junegunn/fzf#370]
`:Map` (`:FzfMap` for me) command fails. When I run the command manually in the shell, it reports a panic:

```
$ cat "/var/folders/hg/wjh_7mvn1l32gjrgmvhhs7xw0000gp/T/nvimCVBeaa/8"|/Users/justin/.fzf/bin/fzf --prompt "Maps (n)> " --ansi --no-hscroll --nth 1,.. --color prompt:12 > /var/folders/hg/wjh_7mvn1l32gjrgmvhhs7xw0000gp/T/nvimCVBeaa/7

panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x40450c5]

goroutine 1 [running]:
github.com/junegunn/fzf/src.parseTheme(0x0, 0x7fff5fbff996, 0x9, 0x7)
        /Users/jg/gosrc/src/github.com/junegunn/fzf/src/options.go:388 +0xae5
github.com/junegunn/fzf/src.parseOptions(0xc208060000, 0xc20800a010, 0x8, 0x8)
        /Users/jg/gosrc/src/github.com/junegunn/fzf/src/options.go:704 +0x2666
github.com/junegunn/fzf/src.ParseOptions(0x40021b2)
        /Users/jg/gosrc/src/github.com/junegunn/fzf/src/options.go:890 +0xfc
main.main()
        /Users/jg/gosrc/src/github.com/junegunn/fzf/src/fzf/main.go:6 +0x1f

goroutine 2 [runnable]:
runtime.forcegchelper()
        /usr/local/Cellar/go/1.4.2/libexec/src/runtime/proc.go:90
runtime.goexit()
        /usr/local/Cellar/go/1.4.2/libexec/src/runtime/asm_amd64.s:2232 +0x1

goroutine 3 [runnable]:
runtime.bgsweep()
        /usr/local/Cellar/go/1.4.2/libexec/src/runtime/mgc0.go:82
runtime.goexit()
        /usr/local/Cellar/go/1.4.2/libexec/src/runtime/asm_amd64.s:2232 +0x1

goroutine 4 [runnable]:
runtime.runfinq()
        /usr/local/Cellar/go/1.4.2/libexec/src/runtime/malloc.go:712
runtime.goexit()
        /usr/local/Cellar/go/1.4.2/libexec/src/runtime/asm_amd64.s:2232 +0x1

goroutine 5 [runnable]:
os/signal.loop()
        /usr/local/Cellar/go/1.4.2/libexec/src/os/signal/signal_unix.go:19
created by os/signal.init¬∑1
        /usr/local/Cellar/go/1.4.2/libexec/src/os/signal/signal_unix.go:27 +0x35
```

Temporary file which contains the mappings is here: https://gist.github.com/c2065edaab6df6a19e2e

**Comment by junegunn** [comment]
Thanks. Confirmed, will fix it soon.

```sh
fzf +c --color prompt:12
```

(You probably have `--no-color` in your $FZF_DEFAULT_OPTS?)

**Comment by junegunn** [comment]
Should be fixed in fzf 0.10.8. Thanks!
